tests/stats_test.py

```diff
@@ -123,3 +123,106 @@ def test_corr_matrix(self):
         ak.DataFrame(
             {"x": self.x, "y": self.y, "u": self.u, "b": self.b, "f": self.f, "s": self.s, "c": self.c}
         ).corr()
+
+    def test_divmod(self):
+        # vector-vector cases
+        # int int
+        ak_div, ak_mod = ak.divmod(self.x, self.y)
+        np_div, np_mod = np.divmod(self.npx, self.npy)
+        self.assertListEqual(ak_div.to_list(), np_div.tolist())
+        self.assertListEqual(ak_mod.to_list(), np_mod.tolist())
+
+        # int float
+        ak_div, ak_mod = ak.divmod(self.x, ak.cast(self.y, ak.float64))
+        np_div, np_mod = np.divmod(self.npx, self.npy.astype(float))
+        self.assertListEqual(ak_div.to_list(), np_div.tolist())
+        self.assertListEqual(ak_mod.to_list(), np_mod.tolist())
+
+        # float int
+        ak_div, ak_mod = ak.divmod(ak.cast(self.x, ak.float64), self.y)
+        np_div, np_mod = np.divmod(self.npx.astype(float), self.npy)
+        self.assertListEqual(ak_div.to_list(), np_div.tolist())
+        self.assertListEqual(ak_mod.to_list(), np_mod.tolist())
+
+        # float float
+        ak_div, ak_mod = ak.divmod(ak.cast(self.x, ak.float64), ak.cast(self.y, ak.float64))
+        np_div, np_mod = np.divmod(self.npx.astype(float), self.npy.astype(float))
+        self.assertListEqual(ak_div.to_list(), np_div.tolist())
+        self.assertListEqual(ak_mod.to_list(), np_mod.tolist())
+
+        # scalar-vector cases
+        # int int
+        ak_div, ak_mod = ak.divmod(30, self.y)
+        np_div, np_mod = np.divmod(30, self.npy)
+        self.assertListEqual(ak_div.to_list(), np_div.tolist())
+        self.assertListEqual(ak_mod.to_list(), np_mod.tolist())
+
+        # int float
+        ak_div, ak_mod = ak.divmod(30, ak.cast(self.y, ak.float64))
+        np_div, np_mod = np.divmod(30, self.npy.astype(float))
+        self.assertListEqual(ak_div.to_list(), np_div.tolist())
+        self.assertListEqual(ak_mod.to_list(), np_mod.tolist())
+
+        # float int
+        ak_div, ak_mod = ak.divmod(30.0, self.y)
+        np_div, np_mod = np.divmod(30.0, self.npy)
+        self.assertListEqual(ak_div.to_list(), np_div.tolist())
+        self.assertListEqual(ak_mod.to_list(), np_mod.tolist())
+
+        # float float
+        ak_div, ak_mod = ak.divmod(30.0, ak.cast(self.y, ak.float64))
+        np_div, np_mod = np.divmod(30.0, self.npy.astype(float))
+        self.assertListEqual(ak_div.to_list(), np_div.tolist())
+        self.assertListEqual(ak_mod.to_list(), np_mod.tolist())
+
```

Comment:
for the float cases (at the very least both float) we should consider doing something like `self.f + 0.5` instead to have a non-round float